SPECS-EXTENDED/libguestfs/libguestfs.spec

```diff
@@ -809,10 +771,18 @@ fi
 mv README README.orig
 sed 's/@VERSION@/%{version}/g' < %{SOURCE4} > README
 
+mkdir cachedir repo
+find /var/cache/{dnf,tdnf} -type f -name '*.rpm' -print0 | \
+  xargs -0 -n 1 cp -t repo
+createrepo_c repo
+sed -e "s|@PWD@|$(pwd)|" %{SOURCE6} > yum.conf
+
 
 %build
 %{configure} \
   PYTHON=%{__python3} \
+  --with-distro=REDHAT \
```

Comment:
Why is distro "REDHAT"? Is it just to trigger a working config?

Reply:
I'll add a comment for this. This is to indicate that we are "fedora-like" and follow their naming conventions for packages.

Reply:
Comment added.